platform-server-rest/src/main/java/uno/cod/platform/server/rest/controller/ChallengeController.java

```diff
@@ -21,17 +22,20 @@ public ChallengeController(ChallengeService service) {
     }
 
     @RequestMapping(value = RestUrls.CHALLENGES, method = RequestMethod.POST)
+    @PreAuthorize("isAuthenticated() and @securityService.isOrganizationAdmin(principal, #dto.organizationId)")
```

Comment:
because users can be organization members. eg. I want to start x challenge. I`ll be an organization member there. But this does not mean that i can see all the tasks. Isn`t this the thing with organizations?